{"name": "登录账号错误", "status": "passed", "description": "登录失败用例", "attachments": [{"name": "stdout", "source": "2462c269-a3e6-440d-9ca9-cf00c02f3775-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "user", "value": "'aaxxxx01'"}, {"name": "psw", "value": "'123456'"}, {"name": "title", "value": "'登录账号错误'"}], "start": 1639015539517, "stop": 1639015539746, "uuid": "9f938da5-a9eb-4acd-911a-70a0c9cc1327", "historyId": "8ea5593a0a443cf97841c89113ef0174", "testCaseId": "04e9502b72231c99b3e3fd3a8c6be39d", "fullName": "test_case.test_login.Test_login#test_login_fail", "labels": [{"name": "feature", "value": "登录"}, {"name": "story", "value": "登录失败用例"}, {"name": "parentSuite", "value": "test_case"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "Test_login"}, {"name": "host", "value": "v_qangchen-PC1"}, {"name": "thread", "value": "70196-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.test_login"}]}